src/Shell/Helper/ProgressHelper.php

```diff
@@ -115,7 +115,7 @@ public function init(array $args = [])
      * @param int $num The amount of progress to advance by.
      * @return $this
      */
-    public function increment($num = 1)
+    public function increment(int $num = 1)
```

Comment:
This could be a float too. I know we don't have it documented that way, but it could happen and I think we should be ok with a float.

Reply:
I can fix the docblock